source/documentation/administration_guide/appe-enabling_FIPS.adoc

```diff
@@ -29,22 +29,18 @@ FIPS mode is enabled.
 ////
 [IMPORTANT]
 ====
-Red Hat recommends installing Red Hat Enterprise Linux 8 with FIPS mode enabled as opposed to enabling FIPS mode later. Enabling FIPS mode during the installation ensures that the system generates all keys with FIPS-approved algorithms and continuous monitoring tests in place.
+The FIPS mode needs to be enabled before RHV host deploy for Clusters with FIPS enabled. Enabling FIPS mode during the installation ensures that the system generates all keys with FIPS-approved algorithms and continuous monitoring tests in place.
```

Comment:
Please use `{virt-product-shortname}` instead of `RHV`
Also in the other parts of this patch.

Reply:
Done